.github/workflows/update_iers.yml

```diff
@@ -0,0 +1,31 @@
+name: Auto-update IERS tables
+
+on:
+  schedule:
+    - cron: '0 0 * * 0'
+
+jobs:
+  update-iers:
+    name: Auto-update IERS tables
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout code
+      uses: actions/checkout@v2
+    - name: Download latest IERS files
+      run: ./update_builtin_iers.sh
+      working-directory: astropy/utils/iers/data
+    - name: Commit changes
+      run: |
+        git config user.name 'Astrobot'
+        git config user.email 'tom@chi-squared.org'
+        git add astropy/utils/iers/data/Leap_Second.dat
+        git add astropy/utils/iers/data/eopc04_IAU2000.62-now
+        git commit -m "Update IERS Earth rotation and leap second tables"
+    - name: Create Pull Request
+      uses: peter-evans/create-pull-request@v3
+      with:
+        branch: update-iers-
+        branch-suffix: timestamp
+        labels: backport-v5.0.x, merge-when-ci-passes, no-changelog-entry-needed, utils.iers
```

Comment:
So in the release procedure, there needs to be reminder to update the label here every LTS release?

And can we set the milestone automatically too?

Reply:
Well for milestone the issue is that would need to be updated quite often as it would be different for each minor release. I'm tempted to also remove the backport labelling because otherwise it's yet another manual step during releases. Maybe it's enough to rely on maintainers labelling/milestoning the PR? (aside from the other non-backport labels above).